### PR TITLE
Create WinsonChen.txt

### PR DESCRIPTION
**Pull request by WinsonChen114**
Added introduction file for Winson Chen

### CODE REVIEW

**Review by wisabel0 — APPROVED** [review]

**Review by sashreek1 — APPROVED** [review]

File: introductions/WinsonChen.txt [hunk 1]
```diff
@@ -0,0 +1,11 @@
+Name: Winson Chen
+Pronouns: he/him
+Year: Junior
+Major: Computer Engineering
+
+I learned C and C++ from my classes at the University of Michigan.
+I also learned the PERN stack during my internship at CodePath during the summer of 2022.
```
Comment (tabiosg) [inline introductions/WinsonChen.txt:7]: Great work!

File: introductions/WinsonChen.txt [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+Outside of school, I enjoy watching anime and playing Genshin Impact.
```
Comment (tabiosg) [inline introductions/WinsonChen.txt:9]: Wow, seems like we have similar interests!

**Review by tabiosg — APPROVED** [review]

File: introductions/WinsonChen.txt (hunk 2 above)
Comment (WinsonChen114) [reply introductions/WinsonChen.txt:9]: What a coincidence!

File: introductions/WinsonChen.txt (hunk 1 above)
Comment (WinsonChen114) [reply introductions/WinsonChen.txt:7]: Thanks!

**Review by wilsonchen02 — APPROVED** [review]
i ate your recipe. wwoaoa

**Pull request merged**